draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -130,24 +130,30 @@ For any host ICE candidate gathered by a browsing context as part of {{RFC8445}}
 
```

Comment:
I tend to think this section is too prescriptive. Browsers may generate names at any point in time, not just during candidate gathering.

Reply:
True, implementations might choose to preregister some names beforehand, for instance at peer connection creation time.
This might for instance leave enough time to fix any potential name conflict before actually exposing the mdns name through candidates.

Reply:
Loosened the description a little.

Reply:
I tend to think we should completely disconnect the registration text from 8445 gathering. Instead, we could have separate sections for mDNS Registration and ICE Candidate Gathering, and we could simply note that registration could either be done ahead-of-time or on-demand.

When gathering ICE candidates, replace the IP addresses of any gathered host candidates with the mDNS hostname that has been registered for that IP address. This registration may be done ahead of time, or on-demand.

Reply:
I think we should have steps 1-2 and 3-5 in separate sections.